lib/pages/settings_dashboard/settings_security/settings_security_view.dart

```diff
@@ -33,14 +33,15 @@ class SettingsSecurityView extends StatelessWidget {
                 title: Text(L10n.of(context)!.ignoredUsers),
                 onTap: () => context.go('/ignorelist'),
               ),
-              ListTile(
-                leading: const Icon(Icons.password_outlined),
-                trailing: const Icon(Icons.chevron_right_outlined),
-                title: Text(
-                  L10n.of(context)!.changePassword,
-                ),
-                onTap: controller.changePasswordAccountAction,
-              ),
+              // #869 Hide change password for now
+              // ListTile(
+              //   leading: const Icon(Icons.password_outlined),
+              //   trailing: const Icon(Icons.chevron_right_outlined),
+              //   title: Text(
+              //     L10n.of(context)!.changePassword,
+              //   ),
+              //   onTap: controller.changePasswordAccountAction,
+              // ),
```

Comment:
can you help us to hide all the `Privacy & Security` settings?

Reply:
done except this part 
![image](https://github.com/linagora/twake-on-matrix/assets/48354990/fb91b83c-539e-4d2d-98b9-b986d69c24cd)

Reply:
Ok thank you